src/main/java/mezz/jei/ingredients/IngredientSorter.java

```diff
@@ -189,19 +189,19 @@ private static int getMaxDamage(ItemStack itemStack) {
 			maxDamage = itemStack.getMaxDamage();	
 		}
 		return maxDamage;
-	};
+	}
 
 	private static int getHarvestLevel(ItemStack itemStack) {		
 		if (itemStack != ItemStack.EMPTY) {
 			return (itemStack.getToolTypes().size() > 0 ? itemStack.getHarvestLevel(itemStack.getToolTypes().iterator().next(), null, null) : -1);
 		}
 		return -1;
-	};
+	}
```

Comment:
Looking closely, this class is pretty broken. I'll fix this logic up separately after the cleanup here.

Reply:
When you are doing so it may be wroth keeping in mind that the tooltype system is going away in its current form relatively soon https://github.com/MinecraftForge/MinecraftForge/pull/7970

Reply:
Good to know, thanks! I just completed a quick refactor on the sorting logic, so it shouldn't be too bad to update it when that PR is merged.